README.md

```diff
@@ -116,6 +112,15 @@ Additional resources:
 * [serverfull](https://github.com/asecurityteam/serverfull)
 * [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
 
+### Gateway Environment Variables
+Here are some environment variables that need to be set, we recommend using the transportd library to handle authentication on outbound requests to Nexpose
+These environment variables can be named to whatever, however api.yaml(which uses transportd) reflects these names in handling basic authentication
```

Comment:
suggestion: `api.yaml(which uses transportd)` is better said `api.yaml (which is used by transportd)`. The file itself doesn't use anything but transportd does use the spec as configuration.

Reply:
done!